Dockerfile.sle15

```diff
@@ -1,4 +1,5 @@
-FROM registry.suse.de/devel/pubcloud/productionservices/pint/containers/pint/pint-server-lambda-function:latest
+#FROM registry.suse.de/devel/pubcloud/productionservices/pint/containers/pint/pint-server-lambda-function:latest
```

Comment:
Potentially this could be handled with an ARG setting that defaults to the standard path, that can be overridden via a command line argument when building the image to point to the alternate path.

However the value landed by the PR should use the standard path, not a developer's personal path.